setup.cfg

```diff
@@ -91,9 +91,12 @@ boto3 =
     boto3>=1.7.0,<2.0
 
 docs =
-    sphinx>=4.0,<5.0
-    sphinx-argparse>=0.2,<0.3
-    sphinx_rtd_theme
+    mkdocs>=1.5.3
+    mkdocs-material>=9.4.14
+    mkdocstrings>=0.24.0
+    mkdocstrings-python>=1.7.5
+    termynal>=0.11.1
```

Comment:
Feel free to remove [`termynal`](https://github.com/ines/termynal) from the build if it's not utilized.

It's fun, though 😄

Reply:
That does look fun, that would add some dynamic-ness to our docs.

Reply:
I think I do want to leave it in. I'm not using it for anything yet, but it'll be nice to show for things like file transfers.